repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/serializer/InspectorSerializer4Bears.java

```diff
@@ -43,12 +42,15 @@ public InspectorSerializer4Bears(List<SerializerEngine> engines) {
         String typeOfFailures = StringUtils.join(jobStatus.getFailureNames(), ",")+"";
         String previousBuildSlug = (previousBuild != null) ? previousBuild.getRepository().getSlug() : "";
 
+        String committerEmail = (build.getCommit().getCommitterEmail() != null) ? build.getCommit().getCommitterEmail() : "-";
+
         List<Object> dataCol = new ArrayList<Object>();
         dataCol.add(build.getId() + "");
         dataCol.add(previousBuildId + "");
         dataCol.add(inspector.getBuildToBeInspected().getStatus().toString());
         dataCol.add(state);
         dataCol.add(realState);
+        dataCol.add(inspector.getCheckoutType().name());
```

Comment:
You should also add the new information that interest you in the `serializeAsJson` method: the data are consolidated inside the mongoDB database too.